easybuild/easyconfigs/i/IntelClusterChecker/IntelClusterChecker-2017.1.016.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = "IntelBase"
+
+name = 'IntelClusterChecker'
+version = '2017.1.016'
+
+homepage = 'https://software.intel.com/en-us/intel-cluster-checker'
+description = """Verifies cluster components work together ― for 
+ - better uptime and productivity and lower total
+ - cost of ownership (TCO)
+ """
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+sources = ['l_clck_p_%(version)s.tgz']
+checksums = ['7a89553dd0bce18d6c17b14f94d669ffeef8621e01b0a135292fb7ce71fc6e25']
```

Comment:
@shahzebsiddiqui Hmm, I'm getting a different SHA256 checksum: `72fcf16e220dfc1ee1e16a633381f7957e15b0b6f322f9954d1de2a1ec3851f1`.

The MD5sum is the same you had before though (`84e7c00bf1f9b87e248dd62e4df7e3b7 `)...

Reply:
okay let me fix not sure why there is a discrepancy in this. I downloaded this a long time ago